### ISSUE TITLE
DE Australia - Jun-Sep 2020 migration to MobX

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/de-australia-map#60]
- [x] catalogue - Checklist to test the upgraded http://de-australia.saas.terria.io/ matches the features and performance of http://de-australia.terria.io/

- [x] help menu 

- [x] https not http

- [ ] url redirect to http://maps.dea.ga.gov.au/

- [x] ~double check~ update disclaimer to match ~- where to; because it will be replaced by welcome onboarding modal ~

- [x] catalogue needs to match v7

- [x] v7 share links need to be maintained

Examples to test: 
- https://www.ga.gov.au/dea/products/iem
- https://www.ga.gov.au/dea/products/hltc

SaaS
![image](https://user-images.githubusercontent.com/18452161/91685562-0cc7c400-eb9e-11ea-8193-3042717d2ad1.png)

v7
![image](https://user-images.githubusercontent.com/18452161/91685585-1e10d080-eb9e-11ea-9946-e43b7bcc2108.png)

**Comment by AnaBelgun** [comment]
https://de-australia.dev.saas.terria.io/

**Comment by AnaBelgun** [comment]
Feedback from testing which needs to be incorporated before decommissioning: 

- [ ] waterbodies https://github.com/TerriaJS/terriajs/issues/4852

**Comment by soyarsauce** [comment]
https://github.com/TerriaJS/terriajs/issues/4852

**Comment by soyarsauce** [comment]
https://github.com/TerriaJS/terriajs/issues/4851

**Comment by steve9164** [comment]
Waiting on https://github.com/TerriaJS/terriajs/issues/4774

**Comment by nf-s** [comment]
- [ ] Legend issue https://github.com/opendatacube/datacube-ows/issues/467

**Comment by nf-s** [comment]
Legend issue is done!

For sharekeys progress see https://github.com/TerriaJS/terriajs/issues/4774#issuecomment-737253349

**Comment by nf-s** [comment]
Sharekeys are done!

**Comment by nf-s** [comment]
Deployed to dev cluster - https://de-australia.dev.saas.terria.io/

It currently uses the old v7 catalogs:
- https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/master/dev/terria/dea-maps.json
- https://maps.dea.ga.gov.au/init/terria-cube.json

So any changes to those will update the map

**Comment by AnaBelgun** [comment]
URL sent on 24/12/2020 to DEA team to test and provide feedback

**Comment by AnaBelgun** [comment]
This ticket is superseded by monthly release ticket/s: https://github.com/TerriaJS/de-australia-map/issues/74
